platform/core/src/services/ToolBarService/ToolBarService.js

```diff
@@ -17,24 +20,78 @@ export default class ToolBarService {
        */
     };
 
+    // TODO: Do we need to track per context? Or do we allow for a mixed
+    // definition that adapts based on context?
+    this.state = {
+      primaryToolId: 'Wwwc',
+      toggles: {
+        /* id: true/false */
+      },
+      groups: {
+        /* track most recent click per group...? */
+      },
+    };
+
     Object.assign(this, pubSubServiceInterface);
   }
 
   init(extensionManager) {
     this.extensionManager = extensionManager;
   }
 
+  /**
+   *
```

Comment:
I feel like this is obscure enough at first glance that fleshing this jsdoc out would help.